Community/NewLocation/templates/NewLocation_page.html

```diff
@@ -0,0 +1,28 @@
+<!-- Copyright 2018 BlueCat Networks. All rights reserved. -->
```

Comment:
Please change date to 2019.

Reply:
You caught the one I missed! Which is weird because I opened the file and didn't get a hit. Fixed!